firmware/robot2015/src-ctrl/modules/controller/controller.hpp

```diff
@@ -0,0 +1,5 @@
+
+#include "robot-devices.hpp"
+
+
+void Task_Controller(void const*);
```

Comment:
the controller.cpp was where I was going to put functions that will filter the imu readings and/or integrate the LQR stuff into everything somehow. I briefly looked at the LQR stuff, but never really go a grasp at what was going on or how it would be used on-board, so I'm sure you have better judgement than I would on what to do here.